### PR TITLE
Add Actions CI

### PR DESCRIPTION
**Pull request by MSP-Greg**
Run Ubuntu, macOS, Windows, all 2.3 thru head.

### CODE REVIEW

**Comment by MSP-Greg** [comment]
See https://github.com/MSP-Greg/bson-ruby/actions/runs/144074166 for a run in my fork.

I believe if Actions isn't running in a repo, the first PR adding workflows will not trigger a run.

**Comment by p-mongo** [comment]
Hi @MSP-Greg , thank you for the PR.

Can you please add a prose description of what this PR is meant to achieve? We already have testing configured using various CI systems.

File: .github/workflows/bson-ruby.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        || contains(github.event.head_commit.message, '[skip ci]'))
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu, macos, windows ]
+        ruby: [ 2.3, 2.4, 2.5, 2.6, 2.7, head ]
+        include:
+          - { os: windows , ruby: mingw }
+        exclude:
+          - { os: windows , ruby: head }
```
Comment (p-mongo) [inline .github/workflows/bson-ruby.yml:27]: Also why is windows/ruby head excluded? Is this not a possible configuration?

Comment (MSP-Greg) [reply .github/workflows/bson-ruby.yml:27]: @p-mongo

Sorry for the delay, having issues with internet service.

If you want to stick with Travis & AppVeyor, that's fine.  There are pro's and con's to either.  Actions has a much higher limit for concurrent jobs.  One con re Actions is that it has no 32-bit Ruby support on Windows.

As to ruby-head vs ruby-mingw, they're both master builds.  I am the builder/supplier of the mingw and mswin builds.  They're fully tested and used by several repos, including ruby organization repos.  Your choice.

**Pull request merged**